lib/logstash/filters/mutate.rb

```diff
@@ -202,37 +211,36 @@ def register
         :replacement  => replacement
       }
     end
-  end # def register
 
-  public
-  def filter(event)
-    
-
-    rename(event) if @rename
-    update(event) if @update
-    replace(event) if @replace
-    convert(event) if @convert
-    gsub(event) if @gsub
-    uppercase(event) if @uppercase
-    lowercase(event) if @lowercase
-    strip(event) if @strip
-    remove(event) if @remove
-    split(event) if @split
-    join(event) if @join
-    merge(event) if @merge
+    @mutators = []
```

Comment:
Are there any serious performance implications when the predicate check is moved to the method? e.g.
```
def rename(event)
  return unless @rename
  ...
end
```